src/main/java/graphql/ExecutionInput.java

```diff
@@ -0,0 +1,94 @@
+package graphql;
```

Comment:
Rock and roll.  I was thinking the library needed this versus the myriad of execute(....) overlaods it has now